dashboard/app/views/home/_homepage.html.haml

```diff
@@ -48,6 +48,11 @@
     %a{href:'/pd/workshop_dashboard'}
       %h3
         Manage PD Workshops
+    -# Update this to workshop organizer
```

Comment:
nit: prefix with `TODO:` and optionally (preferred) your name in parentheses so we know who's on the hook

Reply:
And you mean `workshop_organizer` OR `workshop_admin` right?

Reply:
Yes to both